eth/backend.go

```diff
@@ -229,16 +229,17 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 		checkpoint = params.TrustedCheckpoints[genesisHash]
 	}
 	if eth.handler, err = newHandler(&handlerConfig{
-		Database:       chainDb,
-		Chain:          eth.blockchain,
-		TxPool:         eth.txPool,
-		Merger:         merger,
-		Network:        config.NetworkId,
-		Sync:           config.SyncMode,
-		BloomCache:     uint64(cacheLimit),
-		EventMux:       eth.eventMux,
-		Checkpoint:     checkpoint,
-		RequiredBlocks: config.RequiredBlocks,
+		Database:        chainDb,
+		Chain:           eth.blockchain,
+		TxPool:          eth.txPool,
+		DisableTxGossip: config.RollupDisableTxPool,
```

Comment:
Can you move this to the end of the struct fields, to match the location of the value?
And maybe rename it to `NoTxGossip` instead of `DisableTxGossip`, then it's shorter than the `RequiredBlocks` field name and we don't have to reformat the whole that way, minimizing the diff nicely.

Reply:
Yep, much better, thanks!